Makefile

```diff
@@ -127,3 +127,12 @@ test-cover:
 	@echo "--> Generating coverage.txt"
 	@export VERSION=$(VERSION); bash -x scripts/test_cover.sh
 .PHONY: test-cover
+
+## adr-gen: Download the adr template from the celestiaorg/.github repo and
+#create a new adr file with the provided adr number and title.
+#
+# Usage: make adr-gen NUM=016 TITLE=my-adr-title
```

Comment:
The `make help` command doesn't print this correctly:

```
...
  adr-gen        Download the adr template from the celestiaorg/.github repo and
```

So proposal to summarize this text into one line:
```suggestion
## adr-gen: Download the ADR template from the celestiaorg/.github repo. Ex. `make adr-gen NUM=016 TITLE=my-adr-title`
```